packages/flame/src/Toaster/Toaster.tsx

```diff
@@ -0,0 +1,292 @@
+import * as React from 'react';
+import { keyframes } from '@emotion/core';
+import { css } from '@styled-system/css';
+import {
+  ToastProvider,
+  ToastProviderProps,
+  ToastProps,
+  useToasts as useReactToastNotifications,
+  Options,
+} from 'react-toast-notifications';
+
+import { IconVerified } from '../Icon/Verified';
+import { IconDanger } from '../Icon/Danger';
+import { IconMathMultiply } from '../Icon/MathMultiply';
+import { Box, Flex } from '../Core';
+
+const toastStates = {
+  entering: { transform: 'translate3d(0, 120%, 0)', opacity: 0 },
+  entered: { transform: 'translate3d(0,0,0)', opacity: 1 },
+  exiting: { transform: 'translate3d(0, 120%, 0)', opacity: 0 },
+  exited: { transform: 'translate3d(0, 120%, 0)', opacity: 0 },
+};
```

Comment:
Was multi-stacking animation implemented from specs? There is a slight "jump" after a an extra toaster is shown:

![Kapture 2020-06-08 at 15 20 31](https://user-images.githubusercontent.com/24464151/84071701-280aa500-a99c-11ea-8ad3-44fd668eb8b7.gif)

Reply:
Multi stacking is apparently allowed, but no specs are written.

If anything, users can disable the functionality very easily by checking if there are any notifications in the stack.

jump is due to the underlying lib not properly calculating where things should pop, I would need to PR or hack around the issue. Its not as jarring as their own example luckily.